docs/source/index.rst

```diff
@@ -5,80 +5,100 @@
 
 **aiida-core version:** |release|
 
-.. figure:: images/AiiDA_transparent_logo.png
+.. image:: images/AiiDA_transparent_logo.png
     :width: 250px
     :align: center
+    :alt: AiiDA logo
+    :class: mb-3
 
-    Automated Interactive Infrastructure and Database for Computational Science
-
-#################################
-Welcome to AiiDA's documentation!
-#################################
+###########################################################################
+Automated Interactive Infrastructure and Database for Computational Science
```

Comment:
Yeh thats fine, I only added it here because I removed the long name as a figure caption, since it was annoying me that it was starting with `fig 1`. So you are happy for "Automated Interactive Infrastructure and Database for Computational Science" not to appear anywhere on the page?

Reply:
I am, but we should certainly seek team input on this. Maybe let's remove it for now. I'll invite everyone to review the final docs anyways before I merge them so we can see if this is a problem.

Reply:
I also agree this should be removed

Reply:
fixed